fury/animation/interpolator.py

```diff
@@ -0,0 +1,348 @@
+import numpy as np
+from scipy import interpolate
+from scipy.spatial import transform
+from fury.colormap import rgb2hsv, hsv2rgb, rgb2lab, lab2rgb, xyz2rgb, rgb2xyz
+
+
+class Interpolator:
```

Comment:
I've talked to @skoudoro about switching from Interpolator classes to using closures, but now looking at the code there are extra helper methods that need access to the cached data. So let's discuss this and make a decision in the next meting.